recipe/patch_yaml/conda-smithy.yaml

```diff
@@ -19,3 +19,11 @@ then:
   - tighten_depends:
       name: conda
       upper_bound: "5"
+if:
+  name: conda-smithy
+  version: 3.30.1
+then:
+  - replace_depends:
+    old: conda-build?( *)
+    new: conda-build<3.28.2
```

Comment:
This would remove the lower bounds, right?

Reply:
Good catch. Should be fixed now.